plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestConstraintExtractor.java

```diff
@@ -55,8 +56,8 @@
 import static com.google.common.collect.ImmutableMap.toImmutableMap;
 import static io.airlift.slice.Slices.utf8Slice;
 import static io.trino.SessionTestUtils.TEST_SESSION;
+import static io.trino.plugin.base.filter.UtcConstraintExtractor.extractTupleDomain;
```

Comment:
move and rename TestConstraintExtractor too when moving/renaming the tested class